src/components/Requirements.vue

```diff
@@ -596,12 +624,32 @@ button.view {
   color: #757575;
 
   &-progress {
-    font-weight: bold;
     font-size: 12px;
     line-height: 12px;
   }
 }
 
+.completed {
+  margin-top: 1rem;
+  &-ptext {
+    font-size: 12px;
+    opacity: 0.8;
```

Comment:
For completed reqs, are the descriptions supposed to be the same as normal reqs, but just the titles are smaller with less opacity? Looks a bit off to me.

Reply:
Shot a msg to Shane in the cp-designs channel

Reply:
Shane got back to me about this and yes the description should be like the completed req titles and have smaller font and lower opacity